### PR TITLE
feat: activityStreams option

### PR DESCRIPTION
**Pull request by ahammond**
Fixes #

### CODE REVIEW

File: src/aurora.activity-stream.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+interface CreateData {
+  PhysicalResourceId: string;
+}
+
+interface CreateResultProps {
+  Data?: CreateData;
+  PhysicalResourceId: string;
+  ReasonPrefix?: string;
+  Status: CfnStatus;
+}
```
Comment (Justinon) [inline src/aurora.activity-stream.ts:47]:
```suggestion
interface CreateResultProps extends DeleteResultProps {
  PhysicalResourceId: string;
  Status: CfnStatus;
}
```

Comment (Justinon) [reply src/aurora.activity-stream.ts:47]: Or something like that. You get the idea.

File: src/aurora.activity-stream.ts [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+    const rds = new awsSdk.RDS();
+    const result = await rds
+      .startActivityStream({
+        ResourceArn,
+        KmsKeyId: event.ResourceProperties.kmsKeyId,
+        Mode: 'async',
+        ApplyImmediately: true,
+      })
+      .promise();
```
Comment (Justinon) [inline src/aurora.activity-stream.ts:116]: Does a call to `.promise()` need to happen here given the `await` keyword?

File: test/aurora.activity-stream.on-event.test.ts [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    RequestType: 'Update',
+    ResourceProperties: {
+      ServiceToken: 'fakeServiceToken',
+      clusterId: 'aurora-cluster-1',
+      kmsKeyId: 'fakeKmsKeyId',
+    },
+  };
+
+  // I mean... this should never happen, but...
+  it('fails when onUpate fails', async () => {
```
Comment (Justinon) [inline test/aurora.activity-stream.on-event.test.ts:107]:
```suggestion
  it('fails when onUpdate fails', async () => {
```

File: test/aurora.test.ts [hunk 4]
```diff
@@ -42,6 +42,17 @@ describe('Aurora', () => {
     // });
   });
   describe('options', () => {
+    it('activityStream', () => {
```
Comment (Justinon) [inline test/aurora.test.ts:45]: These aren't very descriptive. Maybe:
```suggestion
  describe('When set, option:', () => {
    it('activityStream creates an RDSActivityStream', () => {
```
Somethin' like that.

**Review by Justinon — APPROVED** [review]
LGTM.

File: test/aurora.test.ts (hunk 4 above)
Comment (ahammond) [reply test/aurora.test.ts:45]: rewording the tests to move towards Literate Testing isn't a bad idea, but it is outside the scope of this PR.

File: src/aurora.activity-stream.ts (hunk 2 above)
Comment (ahammond) [reply src/aurora.activity-stream.ts:116]: Yes. We need to `.promise()` in order to get the actual result.

**Pull request merged**